{{cookiecutter.__src_folder_name}}/.devcontainer/devcontainer.json

```diff
@@ -74,17 +57,17 @@
         "ghcr.io/azure/azure-dev/azd:latest": {}
     },
     {% if cookiecutter.project_backend == "django" %}
-    "postCreateCommand": "python -m pip install -r requirements-dev.txt && python src/manage.py migrate && python src/manage.py loaddata src/seed_data.json"
+    "postCreateCommand": "playwright install chromium --with-deps && python src/manage.py migrate && python src/manage.py loaddata src/seed_data.json"
```

Comment:
This does mean that playwright deps won't be in the pre-build right? I guess just chromium is a lot faster, but maybe we should still have an issue to try to move `playwright install chromium` into Dockerfile in the future.